src/main.cpp

```diff
@@ -516,6 +517,10 @@ static void sigaction_signal(int signum, void (*handler)(int)) {
 int main(int argc, char* argv[]) {
     Globals g;
     parse_arguments(argc, argv, g);
+
+    if(!setlocale(LC_CTYPE, "") || !XSupportsLocale()) {
+        fputs("warning: no locale support\n", stderr);
```

Comment:
Why fputs when in the the rest of the main we use std::cerr?

Reply:
Because I've stole the line from dwm :'-)... It's fixed now.

Reply:
I see :D Thanks :)